packages/f-card/src/components/Card.vue

```diff
@@ -107,6 +107,7 @@ $card--pageContentWrapper-width           : 460px;
         transition: 250ms padding ease-in-out;
         padding-left: 6%;
         padding-right: 6%;
+        margin: spacing() * 5 0;
```

Comment:
I think 
```suggestion
        margin: spacing(x5) 0;
```
would be more idiomatic?

Reply:
Will give it a go, just copied the syntax in the rest of the file :D